public/locales/en/translation.json

```diff
@@ -457,6 +457,8 @@
         "addgamestosteam": "Add games to Steam automatically",
         "alt-gogdl-bin": "Choose an Alternative GOGDL Binary to use",
         "alt-legendary-bin": "Choose an Alternative Legendary Binary",
+        "auto-close-launcher": "Close Heroic After Game Launch ⚠️ Disables Cloud Async",
+        "auto-close-launcher-description": "Enabling this setting will disable usage of cloud async but will slightly improve performance",
```

Comment:
I would't say the `will slightly improve performance` part, that will bias people to enable this even if not really needed. I don't think the performance gain of closing heroic will be noticeable in most cases, it might be a bit noticeable in really specific cases with limited resources maybe but it's a maybe.

Reply:
Yeah good point